src/Polytopes/Cone/properties.jl

```diff
@@ -4,20 +4,9 @@
 ###############################################################################
 ###############################################################################
 
-struct ConeRayIterator
-    cone::Cone
-end
-
-function Base.iterate(iter::ConeRayIterator, index = 1)
-    rays = pm_cone(iter.cone).RAYS
-    if size(rays, 1) < index
-        return nothing
-    end
+rays(as::Type{RayVector{T}}, C::Cone) where T = VectorIterator{as}(C.pm_cone.RAYS)
```

Comment:
FYI I think you can also write this (but I am not asking you to change the PR, just wanted to make sure you are aware -- perhaps you already are)
```suggestion
rays(as::Type{<:RayVector}, C::Cone) = VectorIterator{as}(C.pm_cone.RAYS)
```